edweiss-app/__test__/showtime/LectureScreen.test.tsx

```diff
@@ -264,19 +276,11 @@ describe('LectureScreen Component', () => {
         (useUser as jest.Mock).mockReturnValue({ user: { name: 'Test User', }, });
     });
 
-    it('updates UI when new questions are added dynamically', () => {
-        // Verifies that the component re-renders when new question data is added dynamically
-        const { rerender } = render(<LectureScreen />);
-        (useDynamicDocs as jest.Mock).mockReturnValueOnce([...mockQuestionData, { id: '2', data: { text: 'New Question' } }]);
-        rerender(<LectureScreen />);
-        expect(screen.getByText('New Question')).toBeTruthy();
-    });
-
-    it('calls setLandscape on mount', async () => {
+    it('calls setPortrait on mount', async () => {
         // Ensures that the screen is locked in landscape mode when the component mounts
         const mockLockAsync = jest.spyOn(ScreenOrientation, 'lockAsync');
         render(<LectureScreen />);
-        expect(mockLockAsync).toHaveBeenCalledWith(ScreenOrientation.OrientationLock.LANDSCAPE);
+        expect(mockLockAsync).toHaveBeenCalledWith(ScreenOrientation.OrientationLock.PORTRAIT);
```

Comment:
Yes you did a great job at checking that it remains in portrait mode as we discussed before. This comes easier for a user that was beforehand navigating in portrait mode to do not have to experience the rotation adjustments.

Reply:
Yes indeed, it is a better default value then Landscape